pxr/imaging/hdx/colorCorrectionTask.cpp

```diff
@@ -877,11 +877,19 @@ HdxColorCorrectionTask::_CreateBufferResources()
     }
 
     // A larger-than screen triangle made to fit the screen.
-    constexpr float vertData[][6] =
+    float vertData[][6] =
             { { -1,  3, 0, 1,     0, 2 },
               { -1, -1, 0, 1,     0, 0 },
               {  3, -1, 0, 1,     2, 0 } };
 
+    // Vulkan back-end needs the UVs inverted along the y axis
```

Comment:
Instead of fixing this here, it would be better to bring in the fix at the shader generation level so it will also fix the selection highlight rendering in one place.  See https://github.com/PixarAnimationStudios/OpenUSD/pull/2504

Reply:
My PR had some issues with HdStorm, I have updated it to remove some shader compilation errors.